examples/task-tracker/client/tasks/add_comment_to_task_parameters.go

```diff
@@ -112,6 +133,16 @@ func (o *AddCommentToTaskParams) SetID(id int64) {
 	o.ID = id
 }
 
+// Validate these params
+func (o *AddCommentToTaskParams) Validate(formats strfmt.Registry) error {
+
+	if err := o.Body.Validate(formats); err != nil {
```

Comment:
That struct should actually have a Validate method, it has 2 required fields, so there is a reason it's not rendering that validate method